File: Problem1/ThreeOrFive.cs
using System;

/*
 * Implements the brute force solution 
 *
 */
public class ThreeOrFive 
{
    private static int MAX_VALUE = 1000;

    static void Main(string[] args)
    {
        int total = 0;

        for (int i = 0; i < MAX_VALUE; i++) 
        {
            if (i % 3 == 0 || i % 5 == 0) 
            {
                total += i;
            }
        }

        Console.WriteLine(total);

    }   

}

File: Problem1/ThreeOrFive.dart
void main() {
  int MAX_VALUE = 1000;
  int total = 0;
  for (int i = 1; i < MAX_VALUE; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      total += i;
    }
  }
  print(total);
}